src/context/layers/layer-data.ts

```diff
@@ -1,23 +1,22 @@
 import { createAsyncThunk, AsyncThunk } from '@reduxjs/toolkit';
 import { DiscriminateUnion, LayerType } from '../../config/types';
 import { Extent } from '../../components/MapView/Layers/raster-utils';
-import { fetchNSOLayerData, NSOLayerData } from './nso';
 
+import { fetchNSOLayerData, NSOLayerData } from './nso';
 import { fetchWCSLayerData, WMSLayerData } from './wms';
-import {
-  fetchGroundstationData,
-  GroundstationLayerData,
-} from './groundstation';
+import { fetchPointLayerData, PointLayerData } from './point_data';
 import { BoundaryLayerData, fetchBoundaryLayerData } from './boundary';
 import { fetchImpactLayerData, ImpactLayerData } from './impact';
+
 import type { CreateAsyncThunkTypes, ThunkApi } from '../store';
 
 type LayerSpecificDataTypes = {
   boundary: BoundaryLayerData;
   wms: WMSLayerData;
   impact: ImpactLayerData;
   nso: NSOLayerData;
-  groundstation: GroundstationLayerData;
+  // eslint-disable-next-line camelcase
```

Comment:
this is ok?

Reply:
I think it's fine for now. The other option would be to parse the definition and update it to transform all keys to camelCase as we are doing in the layer utils.

Ultimately we should probably do a deep transformation of all the keys in the json files right on import.